src/actions/WalletActions.tsx

```diff
@@ -146,7 +146,7 @@ function selectEOSWallet(navigation: NavigationBase, walletId: string, currencyC
           message={sprintf(lstrings.create_wallet_account_unfinished_activation_message, currencyCode)}
           buttons={{ ok: { label: lstrings.string_ok } }}
         />
-      ))
+      )).catch(err => console.warn(err))
```

Comment:
For the `Airship.show` cases, I think we should default to `showError`, because we really don't want this guy to crash. Otherwise the user will be scratching their head about why the UI didn't appear.